multiselectfield/forms/fields.py

```diff
@@ -27,8 +27,12 @@ def __init__(self, *args, **kwargs):
         self.min_choices = kwargs.pop('min_choices', None)
         self.max_choices = kwargs.pop('max_choices', None)
         self.max_length = kwargs.pop('max_length', None)
-        self.flat_choices = kwargs.pop('flat_choices')
+        self.flat_choices = kwargs.pop('flat_choices', None)
         super(MultiSelectFormField, self).__init__(*args, **kwargs)
+
+        if self.flat_choices is None and self.choices:
```

Comment:
if self.flat_choices is None:
     self.flat_choices = list(flatten_choices(self.choices))